.github/workflows/docker.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     env:
       DOCKER_REGISTRY: ghcr.io
-      DOCKER_IMAGE: osmosis-labs/meshd
+      DOCKER_IMAGE: babylonchain/bcd
```

Comment:
Babylon consumer daemon?

I was thinking about something like `consumerd`, but this is fine.

Reply:
Yeah I meant "babylon consumer daemon". Added reference to mesh-security-sdk to README.md as well